pkg/user/handler.go

```diff
@@ -227,10 +225,8 @@ func (h Handler) FindById(c *gin.Context) {
 	//	403: Error
 	//	404: Error
 	//	415: Error
-	idParam := c.Param("id")
-	id, err := strconv.ParseUint(idParam, 10, 32)
-	if err != nil {
-		_ = c.AbortWithError(http.StatusBadRequest, errors.New("error parsing id"))
+	id, ok := handler.GetPathParameter(c, "id")
```

Comment:
type conversion in `userWithGroups, err := h.userService.FindById(uint(id))` is not necessary anymore 😄

Reply:
Yes, that's the whole point! :-)